tools/shared.py

```diff
@@ -952,7 +952,7 @@ def apply_configuration():
 
 # Target choice.
 ASM_JS_TARGET = 'asmjs-unknown-emscripten'
-WASM_TARGET = 'wasm32-unknown-unknown-wasm'
```

Comment:
You can drop the final `-wasm` here too while you are there.

Reply:
Cool, removed `-wasm`.